pkgs/tools/misc/gh-cal/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchCrate
+, rustPlatform
+, pkgconfig
+, openssl
+, Security
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "gh-cal";
+  version = "0.1.3";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-x9DekflZoXxH964isWCi6YuV3v/iIyYOuRYVgKaUBx0=";
+  };
+
+  cargoSha256 = "sha256-73gqk0DjhaLGIEP5VQQlubPomxHQyg4RnY5XTgE7msQ=";
+
+  nativeBuildInputs = [ pkgconfig ];
```

Comment:
How did this pass ofborg? This should be pkg-config.

Reply:
I think `pkg-config` and `pkgconfig` are aliases to each other. Should I create a new PR to fix that?

Reply:
Oops I totally missed this. Please use pkg-config.

Reply:
> I think `pkg-config` and `pkgconfig` are aliases to each other. Should I create a new PR to fix that?

Yes but ofborg should forbid the alias. So please create a PR to fix that.

Reply:
Fixed in #175443.